security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -12,7 +12,25 @@ These release notes track the development of {cert-manager-operator}.
 
 For more information, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
 
-[id="cert-manager-operator-release-notes-1-13"]
+[id="cert-manager-operator-release-notes-1-13-1"]
+== {cert-manager-operator} 1.13.1
+
+Issued: 2024-05-15
+
+The following advisory is available for the {cert-manager-operator} 1.13.1:
+
+* link:https://access.redhat.com/errata/RHEA-2024:2849[RHEA-2024:2849]
+
+Version `1.13.1` of the {cert-manager-operator} is based on the upstream cert-manager version `v1.13.6`. For more information, see the link:https://cert-manager.io/docs/releases/release-notes/release-notes-1.13#v1136[cert-manager project release notes for v1.13.6].
+
+[id="cert-manager-operator-1-13-1-CVEs"]
+=== CVEs
+
+* link:https://access.redhat.com/security/cve/CVE-2023-45288[CVE-2023-45288]
+* link:https://access.redhat.com/security/cve/CVE-2023-48795[CVE-2023-48795]
+* link:https://access.redhat.com/security/cve/CVE-2024-24783[CVE-2024-24783]
```

Comment:
there are 9 CVEs in the advisory. Also, I do not see any common ones either. Is this intentional?

Reply:
I have added the ones that were given by SME.

Reply:
Can you please verify with SME?

Reply:
Seems like some issue with errata, but the ones provided from the swarup are the ones fixed. And swarup mentioned as long as we keep 45288, for the other CVEs, we can decide which ones we need to add.

Reply:
thanks